deployment/bash/frontdoorazuredeploy.json

```diff
@@ -1,250 +0,0 @@
-{
```

Comment:
this is encapsulated in the `arm/global.json` correct?

Reply:
Yes, frontdoor config is in global ARM template.